### PR TITLE
implementing QE comments

### PR DESCRIPTION
**Pull request by klgill**
I implemented the QE comments on the adoption guide from https://docs.google.com/document/d/1f6t_Jkq9jl4xfWytWIkX3FnYPRFir-oPufy3AMESAGg/edit?tab=t.0#heading=h.kg1e9845y8p6

### CODE REVIEW

File: docs_user/modules/proc_verifying-the-image-service-adoption.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ ceph.conf
 . Check that the service is active, and that the endpoints are updated in the {OpenStackShort} CLI:
 +
 ----
-$ oc rsh openstackclient -n openstackclient
```
Comment (fmount) [inline docs_user/modules/proc_verifying-the-image-service-adoption.adoc:45]: +1 from me.

**Review by jistr — APPROVED** [review]

**Review by archana203 — COMMENTED** [review]
LGTM

File: docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc [hunk 2]
```diff
@@ -515,6 +515,8 @@ spec:
   serviceUser: neutron
    ...
       customServiceConfig: |
+          [DEFAULT]
+          dhcp_agent_notification = True
```
Comment (slawqo) [inline docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc:519]: just a tiny, nitty nit: to be consistent you could use "true" (all lowercase) as in L521 or change L521 to be "True", same as here

File: docs_user/modules/proc_migrating-object-storage-data-to-rhoso-nodes.adoc [hunk 3]
```diff
@@ -17,7 +17,7 @@ The {object_storage_first_ref} migration involves the following steps:
 * For DNS servers, ensure that all existing nodes are able to resolve the hostnames of the {rhocp_long} pods, for example, by using the external IP of the DNSMasq service as the nameserver in `/etc/resolv.conf`:
 +
 ----
-$ oc get service dnsmasq-dns -o jsonpath="{.status.loadBalancer.ingress[0].ip}" | CONTROLLER1_SSH tee /etc/resolv.conf
+$ oc get service dnsmasq-dns -o jsonpath="{.status.loadBalancer.ingress[0].ip}" | $CONTROLLER1_SSH sudo tee /etc/resolv.conf
```
Comment (slawqo) [inline docs_user/modules/proc_migrating-object-storage-data-to-rhoso-nodes.adoc:20]: is this "sudo" really needed here?

**Comment by slawqo** [comment]
I added 2 nit comments but You don't need to respin just to address those. LGTM for me in overall

File: docs_user/modules/proc_migrating-object-storage-data-to-rhoso-nodes.adoc (hunk 3 above)
Comment (archana203) [reply docs_user/modules/proc_migrating-object-storage-data-to-rhoso-nodes.adoc:20]: It is needed in cases when non-sudo user is used to perform adoption i.e, ssh to controllers.
[stack@undercloud-0 ~]$ oc get service dnsmasq-dns -o jsonpath="{.status.loadBalancer.ingress[0].ip}" | $CONTROLLER1_SSH tee /etc/resolv.conf
192.168.23.80tee: /etc/resolv.conf: Permission denied

File: docs_user/modules/con_adoption-prerequisites.adoc [hunk 4]
```diff
@@ -27,6 +27,8 @@ Back-up information::
...
+* Perform a minor update to the latest {OpenStackShort} version. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html/performing_a_minor_update_of_red_hat_openstack_platform/index[Performing a minor update of Red Hat OpenStack Platform].
+* If the `systemd-container` package is not installed on your Compute hosts, reboot all hypervisors one by one to install the `systemd-container` package. To avoid interrupting your workloads during the reboot, live migrate virtual machine instances before rebooting a node. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html/performing_a_minor_update_of_red_hat_openstack_platform/assembly_rebooting-the-overcloud_keeping-updated#proc_rebooting-compute-nodes_rebooting-the-overcloud[Rebooting Compute nodes] in _Performing a minor update of Red Hat OpenStack Platform_.
```
Comment (bogdando) [inline docs_user/modules/con_adoption-prerequisites.adoc:31]: lgtm, thanks

**Pull request merged**